changelog.config.js

```diff
@@ -0,0 +1,88 @@
+const COMPONENT_SPECS = require('genesys-spark-components/build/component-specs/component-specs.json');
+const CHART_COMPONENT_SPECS = require('genesys-spark-chart-components/build/component-specs/component-specs.json');
+
+const componentSpecs = Object.assign(COMPONENT_SPECS, CHART_COMPONENT_SPECS);
+
+const getComponentSpec = function () {
+  let scopeList = ['other'];
+  for (const [key] of Object.entries(componentSpecs)) {
+    scopeList.push(key);
+  }
+  return scopeList;
+};
+
+module.exports = {
+  disableEmoji: true,
+  format: '{type}{scope}: {emoji}{subject}',
+  list: [
+    'test',
+    'feat',
+    'fix',
+    'chore',
+    'docs',
+    'refactor',
+    'style',
+    'ci',
+    'perf'
+  ],
+  maxMessageLength: 110,
+  minMessageLength: 3,
+  questions: ['type', 'scope', 'subject', 'body', 'issues'],
+  scopes: getComponentSpec(),
```

Comment:
I like this addition 👍

Reply:
entering an arbitrary string for the scope is not supported in git-cz, so this is the best next thing